src/CONST.ts

```diff
@@ -5404,6 +5404,14 @@ const CONST = {
                 description: `workspace.upgrade.${this.POLICY.CONNECTIONS.NAME.SAGE_INTACCT}.description` as const,
                 icon: 'IntacctSquare',
             },
+            approvals: {
+                id: 'approvals',
```

Comment:
You need to add `as const` to this line to resolve typecheck issues

Reply:
thanks, I was just wondering how to resolve them!